### ISSUE TITLE
:sparkles: Only scroll list when at end of screen

### ISSUE THREAD

**Issue by Xyl-AU** [jurialmunkey/skin.arctic.fuse#387]
### Skin section

Widgets

### Feature description

Currently in a list of episodes, the carousel will start scrolling to the right as soon as you move to the next episode. The focus will still be on the leftmost episode on screen. My request is an option to move the cursor to the rightmost episode on screen before starting to scroll.

### Why is this feature needed? Please justify and explain.

Spoilers. In a list that shows 4 episodes at a time, simply moving to the 2nd episode will reveal the title and thumbnail of the 5th episode. Sometimes these can contain major spoilers. This could be mitigated by only scrolling when actually trying to access the next episode outside of the list, rather than just whenever you scroll.

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.
